rust/src/nfs/nfs.rs

```diff
@@ -647,8 +651,8 @@ impl NFSState {
         }
         if !self.is_udp {
             self.ts_chunk_xid = r.hdr.xid;
-            let file_data_len = w.file_data.len() as u32 - fill_bytes as u32;
-            self.ts_chunk_left = w.file_len as u32 - file_data_len as u32;
+            debug_validate_bug_on!(w.file_data.len() as u32 > w.count);
```

Comment:
I think this will trigger on fuzzing input (even if I have not seen it on real pcaps)
cf code
```
    let (i, count) = be_u32(i)?;
    let (i, stable) = be_u32(i)?;
    let (i, file_len) = be_u32(i)?;
    let (i, file_data) = take(file_len as usize)(i)?;
```

So, count can be arbitrary small compared to w.file_data.len()

Reply:
which path to this do you still see? Record parsing now does
```rust
    let (i, count) = be_u32(i)?;
    let (i, stable) = verify(be_u32, |&v| v <= 2)(i)?;
    let (i, file_len) = verify(be_u32, |&v| v <= count)(i)?;
```
So `file_len <= count`